pynest/examples/gap_junctions_inhibitory_network.py

```diff
@@ -126,7 +126,7 @@
 and connecting the neurons using the `make_symmetric` flag for
 `one_to_one` connections.
 """
-n_connection = n_neuron * gap_per_neuron / 2
+n_connection = n_neuron * gap_per_neuron // 2
```

Comment:
I guess you want to make sure an int is returned even if ``gap_per_neuron`` and ``n_neuron`` are odd... But then I think using ``int(n_neuron * gap_per_neuron / 2)`` might be better since it also works if ``gap_per_neuron`` is a float...